### ISSUE TITLE
New release?

### ISSUE THREAD

**Issue by jbicha** [gtimelog/gtimelog#248]
There have been years of changes since the last tagged release? Are you expecting to do a new release soon?

**Comment by mgedmin** [comment]
Thank you for the query.  I would like to.

The primary reason I haven't yet is burnout.

Secondary reasons are:

- I've replaced popen("/usr/sbin/sendmail") with talking to an SMTP server, but the new email sending code is synchronous and blocks the main thread.  The reason I haven't rewritten it to be asynchronous/threaded is burnout.

- I'm not sure what the release process should be now that gtimelog is on Flathub.  I have a `make release` that reminds me how to prepare a release and upload it to PyPI, but I don't know what exactly I should be doing to update the flatpak.  I could read about it, but burnout.

- There are some pull requests ready (or almost ready) to be merged that I've been neglecting due to burnout.  I feel like I'll disappoint the contributors (those that aren't already disappointed by the neglect) if I make a release without merging them.

Basically I just need for someone to tell me it's okay to tag a release now without having to make it perfect first, and I can do it.

**Comment by jpsnyder** [comment]
Perfection is the enemy of progress. 

- If the email code is in a separate branch, I would say punt it until you feel like working on it, or if someone else wants to pick it up.
- I'm not exactly sure how Flatpak works, but some light googling tells me you just have to update your flathub repo with an updated manifest file: https://github.com/flathub/org.gtimelog.GTimeLog
- There are already lots of nice substantial changes in the unreleased version (https://github.com/gtimelog/gtimelog/blob/master/CHANGES.rst). I don't think anyone would care if their stuff didn't get in yet. You wouldn't want to rush them in anyway.

**Comment by samtuke** [comment]
It's okay to tag a release now without having to make it perfect first. From a gtimelog user of over 10 years.

**Comment by mgedmin** [comment]
Decided to YOLO, ran `make release`, discovered a roadblock: my hacky `distcheck-sdist` Makefile rule discovered that src/gtimelog/locale/ (full of binary compiled translation .mo files) is present in the sdist but absent from the VCS.

I don't remember what the right thing to do is.  I have a Makefile rule to ensure that .mo files are present in the .whl, but they should probably not be present in the sdist?

**Comment by mgedmin** [comment]
0.12.0 is out on PyPI.

I have little idea of what to do to make a Flathub release happen.  I have commit rights to [the repo](https://github.com/flathub/org.gtimelog.GTimeLog/pull/2/files), I think, or at least I should be able to open a PR?

Perhaps @hfiguiere can help?

**Comment by mgedmin** [comment]
Let's see what happens if I do this: https://github.com/flathub/org.gtimelog.GTimeLog/pull/3

**Comment by samtuke** [comment]
Congratulations! And thank you.
